src/storage/index.js

```diff
@@ -13,22 +13,26 @@ const s3 = new aws.S3({
   credentials: config.s3.credentials,
 });
 
-const getFileUpload = (fileTag, maxFileSize) => {
-  const fileUpload = multer({
-    storage: multerS3({
-      s3,
-      bucket: config.s3.bucket,
-      acl: 'public-read',
-      key: (req, file, next) => {
-        next(null, `portal/profiles/${req.user.uuid}${path.extname(file.originalname)}`);
-      },
-    }),
+const uploadToS3 = async (mediaType, file, uuid) => {
```

Comment:
Nit: rename to a generic `upload` since the caller doesn't care (and shouldn't know) where files get stored; it's abstracted away in this service.